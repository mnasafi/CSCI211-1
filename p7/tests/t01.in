echo t01: Using echo to print strings.
echo Each string should be printed on its own line.
echo Strings can contain spaces, tabs, and special characters (!@#$%^&, etc).
echo Spaces <      >, a tab < >.
