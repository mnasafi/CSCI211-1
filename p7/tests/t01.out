t01: Using echo to print strings.
Each string should be printed on its own line.
Strings can contain spaces, tabs, and special characters (!@#$%^&, etc).
Spaces <      >, a tab < >.
